Source/WebKit/UIProcess/RemotePageProxy.cpp

```diff
@@ -192,9 +192,6 @@ void RemotePageProxy::didFailProvisionalLoadForFrame(FrameInfoData&& frameInfo,
     if (!frame)
         return;
 
-    if (auto* provisionalFrame = frame->provisionalFrame(); provisionalFrame && provisionalFrame->isCrossSiteRedirect())
```

Comment:
You can remove `m_isCrossSiteRedirect` from `ProvisionalFrameProxy` after this change.